assets/js/components/ClusterLink.jsx

```diff
@@ -3,6 +3,8 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import classNames from 'classnames';
 
+const KNOWN_TYPES = ['hana_scale_up', 'hana_scale_out', 'ascs_ers'];
```

Comment:
we could move this in `lib/model/index.js` but still, it doesn't change much until we address the architecture and make also agreements about these things :smile: